private-templates-service/adaptivecards-templating-service/src/util/templateutils.ts

```diff
@@ -159,4 +161,18 @@ export function isValidJSONString(input: string) {
 export function parseToken(token: string): string {
   let bearer = token.split(/[ ]+/).pop();
   return bearer || "";
+}
+
+export function createCard(template: JSON, data: JSON): JSON {
+  try {
+    let dataTemplate: ACData.Template = new ACData.Template(template);
+    let context: ACData.EvaluationContext = new ACData.EvaluationContext();
+    context.$root = data;
+    let card: JSON = dataTemplate.expand(context);
+    return card;
+  }
+  catch {
+    return JSON.parse('{}');
```

Comment:
Would it be different than "return {}";

Reply:
I tried to do that, but the type checker wouldn't let. me :(